src/main/java/edu/dbsleipzig/stream/grouping/application/CitiBikeExample.java

```diff
@@ -45,8 +50,10 @@ public class CitiBikeExample {
      * @throws Exception in case of an error
      */
     public static void main(String[] args) throws Exception {
+        Configuration configuration = new Configuration();
```

Comment:
Please remove the config + network setting. The user can do this in Flink config. 
After removing, remember removing imports as well.